src/main/java/jssc/SerialPortList.java

```diff
@@ -330,12 +330,20 @@ public int compare(String valueA, String valueB) {
                     String fileName = file.getName();
                     if(!file.isDirectory() && !file.isFile() && pattern.matcher(fileName).find()){
                         String portName = searchPath + fileName;
-                        long portHandle = serialInterface.openPort(portName, false);//Open port without TIOCEXCL
-                        if(portHandle < 0 && portHandle != SerialNativeInterface.ERR_PORT_BUSY){
-                            continue;
-                        }
-                        else if(portHandle != SerialNativeInterface.ERR_PORT_BUSY) {
-                            serialInterface.closePort(portHandle);
+<<<<<<< HEAD
```

Comment:
This appears to have a merge conflict inline in the code, needs to be resolved before meeting.